package.json

```diff
@@ -22,22 +21,22 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^15.2.9",
-    "@angular/cdk": "^15.2.9",
-    "@angular/common": "^15.2.9",
-    "@angular/compiler": "^15.2.9",
-    "@angular/core": "^15.2.9",
+    "@angular/animations": "^16.1.8",
+    "@angular/cdk": "^16.1.7",
+    "@angular/common": "^16.1.8",
+    "@angular/compiler": "^16.1.8",
+    "@angular/core": "^16.1.8",
     "@ngbracket/ngx-layout": "^15.0.0",
-    "@angular/forms": "^15.2.9",
-    "@angular/material": "^15.2.9",
-    "@angular/platform-browser": "^15.2.9",
-    "@angular/platform-browser-dynamic": "^15.2.9",
-    "@angular/router": "^15.2.9",
-    "@apollo/client": "^3.8.7",
+    "@angular/forms": "^16.1.8",
+    "@angular/material": "^16.1.7",
+    "@angular/platform-browser": "^16.1.8",
+    "@angular/platform-browser-dynamic": "^16.1.8",
+    "@angular/router": "^16.1.8",
+    "@apollo/client": "^3.8.5",
     "@fullstory/browser": "^1.7.1",
     "@hypertrace/hyperdash": "^1.2.1",
-    "@hypertrace/hyperdash-angular": "4.0.0",
-    "@types/d3-hierarchy": "^2.0.5",
+    "@hypertrace/hyperdash-angular": "^5.1.0",
```

Comment:
```suggestion
    "@hypertrace/hyperdash-angular": "^5.2.0",
```

Reply:
can we do it separately? Just in case the new hd version leads to any UI issues.

Reply:
Yeah, that's fine.